lib/matplotlib/_pylab_helpers.py

```diff
@@ -1,37 +1,40 @@
 """
-Manage figures for pyplot interface.
+Manage figures for the pyplot interface.
 """
 
 import atexit
+from collections import OrderedDict
 import gc
 
 
 class Gcf(object):
     """
-    Singleton to manage a set of integer-numbered figures.
-
-    This class is never instantiated; it consists of two class
-    attributes (a list and a dictionary), and a set of static
-    methods that operate on those attributes, accessing them
-    directly as class attributes.
-
-    Attributes:
-
-        *figs*:
-          dictionary of the form {*num*: *manager*, ...}
-
-        *_activeQue*:
-          list of *managers*, with active one at the end
-
+    Singleton to handle a set of "numbered" figures and managers, and keep
```

Comment:
I find the whole docstring a bit unclear. It seems to assume prior knowledge on figures, figure managers and their relation. Would this be a more complete description?

~~~
    Singleton to maintain the relation between figures and their managers, and
    keep track of and "active" figure and manager.

    When created via pyplot, a figure is associated with its own figure
    manager, that handles the interaction between the figure and the backend.

    This association is established via a unique identifier, called
    *figure number* or *manager number*. While figure numbers are often actual
    numbers, they can actually be any hashable value. The number is attached
    to the figure via the `.Figure.number` attribute; and it's mapped to a
    manager in the `.Gcf.figs` dictionary.

    This class is never instantiated; it consists of an `OrderedDict` mapping
    figure/manager numbers to managers, and a set of class methods that
    manipulate this `OrderedDict`.

~~~

Reply:
Actually the number is an attribute of the manager (which is itself an attribute of the canvas), but other than that, looks good.  Will incorporate a slightly edited version.

Reply:
Pushed edited version.  (Note that the identifier does not create the association, it's just used for bookkeeping.)